src/components/Display.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Display({ result }) {
+  return <div>{result}</div>;
+}
+
+Display.defaultProps = {
+  result: 0,
```

Comment:
Should be a string, right?
```suggestion
  result: '0',
```